### ISSUE TITLE
Example, Multiple Animations

### ISSUE THREAD

**Issue by MichaelBarb** [jQueryKeyframes/jQuery.Keyframes#39]
In your example the button "Multiple Animations" works great in Chrome but not in IE 11.  The spin (transform:rotate) animation  is overridden by the roll (transform:rotate).  You never see the higher speed spin in IE.
I doubt if you can do anything about this.  It is a rather minor issue.  You may want to add it to comments in the documentation.
I work with the two browsers Chrome and IE open side by side.  I try everything in both.  Things like this happen all the time.

**Comment by krazyjakee** [comment]
It looks like IE is stripping out inline values that are applied to it.

http://jsfiddle.net/em6dLmjk/

Notice the difference in the two classes that are applied.

I will look into this, thanks very much indeed!

**Comment by krazyjakee** [comment]
Sorry Michael,

By coincidence, I'm working on this issue right now. I think I've cracked it.

**Comment by krazyjakee** [comment]
The issue was here: https://github.com/jQueryKeyframes/jQuery.Keyframes/blob/e21c2a80747a05c50f7606cf3243493f5b188789/example/index.html#L23-L45

In both animations, the effects are modifying the "transform" attribute. This is actually invalid, although chrome handles it anyway. This answer on stack overflow describes it further: http://stackoverflow.com/a/14416948/1013097

I also found other issues in the example which I have no resolved.
Thanks to this issue, I also created a basic debug mode so we can easily see what is being applied to the elements in the console.
